### DISCUSSION TITLE
i18n in react-email

### DISCUSSION THREAD

**Discussion by lowfnm** [resend/react-email#963] [Q&A] [answered]
Could you please share if there is a way to integrate i18n into react-email, if so how do you do it?

**Comment by lowfnm** [comment]
@bukinoshita

**Comment by bobylito** [comment]
I had this question myself and used i18next. More specifically, I pass my email component a translation function. The i18n framework is initialized outside the template.

**Reply by lowfnm** [reply]
is it in your public project, if so can you look for an example?

**Reply by lowfnm** [reply]
I also have a question, we plan to use template as a service that will return ready templates and substitute data from our goland API, is it realizable with react-email?

**Comment by pmioduszewski** [comment, marked as answer]
It's working for me on Vercel like this - [my gist](https://gist.github.com/pmioduszewski/eea40b2564b30e038db08a4b73472fb6)
